Android/app/local.properties

```diff
@@ -0,0 +1,9 @@
+## This file must *NOT* be checked into Version Control Systems,
```

Comment:
Removed the gradle setup under the app folder, since it is actually not required. Unclear why it was added during my current work, perhaps I opened the wrong gradle file at some point and it generated the local properties in the wrong location. 

No matter, they should now be removed.